packages/java/src/browser/java-client-contribution.ts

```diff
@@ -8,44 +8,44 @@
 import { injectable, inject } from "inversify";
 import { CommandService } from "@theia/core/lib/common";
 import {
-    Window, ILanguageClient, BaseLanguageClientContribution, Workspace, Languages, LanguageClientFactory
+  Window, ILanguageClient, BaseLanguageClientContribution, Workspace, Languages, LanguageClientFactory
```

Comment:
Could you check your formatting setting please? It is 4 spaces for ts files.
For VS code we have it checked in: https://github.com/theia-ide/theia/blob/e41a77e57de7f5948e3c61cfe5f5f724b6e2fac1/.vscode/settings.json#L41

Reply:
Naturally I use Theia to develop Theia. I think it is odd that we are depending on VSCode on these.

Reply:
true, but Theia defaults is 4 spaces: https://github.com/theia-ide/theia/blob/f4de4f572bdebd7dadae5e7a3323e39fb346d566/packages/editor/src/browser/editor-preferences.ts#L24

We can check in `.theia/setting.json` as well.